invenio_jobs/services/scheduler.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2024 CERN.
+#
+# Invenio-Jobs is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+import traceback
+import uuid
+from typing import Any
+
+from celery.beat import ScheduleEntry, Scheduler, debug, error, info
+from invenio_db import db
+
+from invenio_jobs.models import Job, Run
+
+
+class JobEntry(ScheduleEntry):
+
+    job = None
+
+    def __init__(self, job, *args, **kwargs):
+        self.job = job
+        super().__init__(*args, **kwargs)
+
+    @classmethod
+    def from_job(cls, job):
+        return cls(
+            job=job,
+            name=job.title,
+            schedule=job.parsed_schedule,
+            args=job.default_args,
+            task=job.task,
+            kwargs={},
```

Comment:
minor: we'll actually have to split `default_args` into `args` and `kwargs` (it matters for Celery tasks). The best way to do so at the moment is by inspecting the task functions (i.e. what's done in the `models.Task.signature` property). I would find a couple of example tasks and see if these splitting logic can be consistently replicated.

Reply:
So we are going with all arguments as kwargs as we don't use positional only arguments as of now. Might be implemented later.